src/index.js

```diff
@@ -1,112 +1,102 @@
-import readPkgUp from 'read-pkg-up';
-import HtmlWebpackIncludeAssetsPlugin from 'html-webpack-include-assets-plugin';
-import ExternalModule from 'webpack/lib/ExternalModule';
-import resolvePkg from 'resolve-pkg';
+const readPkgUp = require('read-pkg-up');
+const ExternalModule = require('webpack/lib/ExternalModule');
+const resolvePkg = require('resolve-pkg');
 
-import getResolver from './get-resolver';
-
-const pluginName = 'dynamic-cdn-webpack-plugin';
-let HtmlWebpackPlugin;
+let HtmlWebpackPlugin = null;
 try {
-    // eslint-disable-next-line import/no-extraneous-dependencies
     HtmlWebpackPlugin = require('html-webpack-plugin');
-} catch (err) {
-    HtmlWebpackPlugin = null;
-}
+// eslint-disable-next-line unicorn/prefer-optional-catch-binding
+} catch (_) {}
 
-const moduleRegex = /^((?:@[a-z0-9][\w-.]+\/)?[a-z0-9][\w-.]*)/;
+const getResolver = require('./get-resolver');
+
+const pluginName = 'dynamic-cdn-webpack-plugin';
+const moduleRegex = /^((?:@[a-z\d][\w-.]+\/)?[a-z\d][\w-.]*)/;
 
 const getEnvironment = mode => {
     switch (mode) {
         case 'none':
         case 'development':
             return 'development';
-
         default:
             return 'production';
     }
 };
 
-export default class DynamicCdnWebpackPlugin {
+module.exports = class DynamicCdnWebpackPlugin {
     constructor({disable = false, env, exclude, only, verbose, resolver} = {}) {
-        if (exclude && only) {
-            throw new Error('You can\'t use \'exclude\' and \'only\' at the same time');
-        }
+        if (exclude && only) throw new Error('You can\'t use \'exclude\' and \'only\' at the same time');
```

Comment:
This formatting change isn't relevant to the PR (and may be opposed to the style the author wants to use)